src/components/NoteLabel/NoteLabel.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { useState } from "react";
+import { useNote } from "../../contexts";
+import "./NoteLabel.css";
+
+export const NoteLabel = () => {
+	const { noteDispatch } = useNote();
+	const [noteLabel, setNoteLabel] = useState("");
+
+	const noteLabelHandler = (e) => {
+		console.log(e.target.name, e.target.value);
+		setNoteLabel(e.target.value.toLowerCase());
+	};
+
+	const setNoteLabelHandler = (e) => {
+		e.preventDefault();
+		console.log(noteLabel);
```

Comment:
Remove console.log statements once debugging is done to keep code and console outputs clean